### PR TITLE
Update README-install.md

### PR DESCRIPTION
**Pull request by sradco**
Updated indentations.

@richm Please review

### CODE REVIEW

**Review by richm — COMMENTED** [review]
Is this extra indentation really necessary?  Markdown says that pre-formatted blocks require only an indentation of 4 spaces, and must begin and end with an empty line: https://daringfireball.net/projects/markdown/syntax#precode

**Comment by sradco** [comment]
When I look at the document text looks large and in some places bolded.
This is the reason I added it.
Do you see the text as expected?

File: README-install.md [hunk 1]
```diff
@@ -67,13 +67,13 @@ For RHEL, you must enable the Extras and the rhel-7-fast-datapath-rpms channels
 ViaQ on Origin requires these [Yum Repos](centos7-viaq.repo).
 You will need to install the following packages: docker, iptables-services.
 
-    # yum install docker iptables-services
+        # yum install docker iptables-services
```
Comment (richm) [inline README-install.md:70]: these look ok now, and do not appear to need extra indentation: https://github.com/ViaQ/Main/blob/master/README-install.md#provisioning-a-machine-to-run-viaq

File: README-install.md [hunk 2]
```diff
@@ -67,13 +67,13 @@ For RHEL, you must enable the Extras and the rhel-7-fast-datapath-rpms channels
...
 
 You will need to configure sudo to not require a tty.  For example, create a
 file like `/etc/sudoers.d/999-cloud-init-requiretty` with the following contents:
 
-    # cat /etc/sudoers.d/999-cloud-init-requiretty
-    Defaults !requiretty
+        # cat /etc/sudoers.d/999-cloud-init-requiretty
+        Defaults !requiretty
```
Comment (richm) [inline README-install.md:76]: these look ok now, and do not appear to need extra indentation: https://github.com/ViaQ/Main/blob/master/README-install.md#provisioning-a-machine-to-run-viaq

File: README-install.md [hunk 3]
```diff
@@ -119,16 +119,16 @@ openshift-ansible-callback-plugins openshift-ansible-filter-plugins
...
 openshift-ansible-roles
 
-    # yum install openshift-ansible \
-      openshift-ansible-callback-plugins openshift-ansible-filter-plugins \
-      openshift-ansible-lookup-plugins openshift-ansible-playbooks \
-      openshift-ansible-roles
+        # yum install openshift-ansible \
+          openshift-ansible-callback-plugins openshift-ansible-filter-plugins \
+          openshift-ansible-lookup-plugins openshift-ansible-playbooks \
+          openshift-ansible-roles
```
Comment (richm) [inline README-install.md:125]: these look ok now, and do not appear to need extra indentation: https://github.com/ViaQ/Main/blob/master/README-install.md#installing-viaq-packages

File: README-install.md [hunk 4]
```diff
@@ -119,16 +119,16 @@ openshift-ansible-callback-plugins openshift-ansible-filter-plugins
...
 
 If the 3.6 version of these packages are not available, you can use the
 git repo `https://github.com/openshift/openshift-ansible.git` and the
 `release-3.6` branch:
 
-    # git clone https://github.com/openshift/openshift-ansible.git -b release-3.6
+        # git clone https://github.com/openshift/openshift-ansible.git -b release-3.6
```
Comment (richm) [inline README-install.md:131]: these look ok now, and do not appear to need extra indentation: https://github.com/ViaQ/Main/blob/master/README-install.md#installing-viaq-packages

File: README-install.md [hunk 5]
```diff
@@ -141,14 +141,14 @@ and which parameters you may want to customize, depending on your environment.
 1. Download the files [vars.yaml.template](vars.yaml.template) and
 [ansible-inventory-origin-36-aio](ansible-inventory-origin-36-aio)
 
-    # curl https://raw.githubusercontent.com/ViaQ/Main/master/vars.yaml.template > vars.yaml.template
-    # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-origin-36-aio > ansible-inventory
+        # curl https://raw.githubusercontent.com/ViaQ/Main/master/vars.yaml.template > vars.yaml.template
+        # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-origin-36-aio > ansible-inventory
```
Comment (richm) [inline README-install.md:145]: This does require the extra indentation

File: README-install.md [hunk 6]
```diff
@@ -141,14 +141,14 @@ and which parameters you may want to customize, depending on your environment.
...
 
 To use ViaQ on Red Hat OCP, use the
 [ansible-inventory-ocp-36-aio](ansible-inventory-ocp-36-aio) file instead
 of the origin-36-aio file (you still need vars.yaml.template):
 
-    # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-ocp-36-aio > ansible-inventory
+        # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-ocp-36-aio > ansible-inventory
```
Comment (richm) [inline README-install.md:151]: This does not need the extra indentation: https://github.com/ViaQ/Main/blob/master/README-install.md#customizing-varsyaml

File: README-install.md [hunk 7]
```diff
@@ -260,11 +260,11 @@ public IP address.
...
-    # cd /usr/share/ansible/openshift-ansible
-    # (or wherever you cloned the git repo if using git)
-    # ANSIBLE_LOG_PATH=/tmp/ansible.log ansible-playbook -vvv \
-      -e @/path/to/vars.yaml \
-      -i /path/to/ansible-inventory playbooks/byo/config.yml
+        # cd /usr/share/ansible/openshift-ansible
+        # (or wherever you cloned the git repo if using git)
+        # ANSIBLE_LOG_PATH=/tmp/ansible.log ansible-playbook -vvv \
+          -e @/path/to/vars.yaml \
+          -i /path/to/ansible-inventory playbooks/byo/config.yml
```
Comment (richm) [inline README-install.md:267]: ok

File: README-install.md [hunk 8]
```diff
@@ -326,20 +327,18 @@ https://github.com/openshift/origin-aggregated-logging/tree/master/hack/kopf
 
 1. To confirm that Elasticsearch, Curator, Kibana, and Fluentd pods are running, run:
 
-    # oc project logging
-    # oc get pods
+        # oc project logging
+        # oc get pods
```
Comment (richm) [inline README-install.md:331]: ok

File: README-install.md [hunk 9]
```diff
@@ -326,20 +327,18 @@ https://github.com/openshift/origin-aggregated-logging/tree/master/hack/kopf
...
 
 2. To confirm that the Elasticsearch and Kibana services are running, run:
 
-    # oc project logging
-    # oc get svc
+        # oc project logging
+        # oc get svc
```
Comment (richm) [inline README-install.md:336]: ok

File: README-install.md [hunk 10]
```diff
@@ -326,20 +327,18 @@ https://github.com/openshift/origin-aggregated-logging/tree/master/hack/kopf
...
 
 3. To confirm that there are routes for Elasticsearch and Kibana, run:
 
-
-    # oc project logging
-    # oc get routes
-
+        # oc project logging
+        # oc get routes
```
Comment (richm) [inline README-install.md:341]: ok

File: README-install.md [hunk 11]
```diff
@@ -443,25 +442,25 @@ To create an "normal" user that can only view logs in a particular set of
 projects, follow the steps above, except do not assign the `cluster-admin`
 role, use the following instead:
 
-    # oc project $namespace
-    # oadm policy add-role-to-user view $username
+        # oc project $namespace
+        # oadm policy add-role-to-user view $username
```
Comment (richm) [inline README-install.md:446]: I don't think these are necessary

File: README-install.md [hunk 12]
```diff
@@ -443,25 +442,25 @@ To create an "normal" user that can only view logs in a particular set of
...
-    # oc create user loguser
-    # oc create identity allow_all:loguser
-    # oc create useridentitymapping allow_all:loguser loguser
-    # oc project ovirt-metrics-engine
-    # oadm policy add-role-to-user view loguser
+        # oc create user loguser
+        # oc create identity allow_all:loguser
+        # oc create useridentitymapping allow_all:loguser loguser
+        # oc project ovirt-metrics-engine
+        # oadm policy add-role-to-user view loguser
```
Comment (richm) [inline README-install.md:458]: I don't think these are necessary

File: README-install.md [hunk 13]
```diff
@@ -443,25 +442,25 @@ To create an "normal" user that can only view logs in a particular set of
...
 
 and to assign the password immediately instead of waiting for the user
 to login:
 
-    # oc login --username=loguser --password=loguser
-    # oc login --username=system:admin
+        # oc login --username=loguser --password=loguser
+        # oc login --username=system:admin
```
Comment (richm) [inline README-install.md:464]: I don't think these are necessary

**Comment by richm** [comment]
I general, the extra indentation is only needed if the pre-code/pre-formatted text is used within a bullet list or numbered list: https://daringfireball.net/projects/markdown/syntax#precode

"One level of indentation — 4 spaces or 1 tab — is removed from each line of the code block."

Lists are indented, so any precode/preformatted blocks in a list must be indented 4 more spaces than the list indentation.

**Pull request closed without merging**